app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -15,8 +15,11 @@
   = t(".dont_cancel", email: @order.distributor.contact.email)
 
 %p &nbsp;
+%h4
+  = t(".order_summary_canceled_html", number: @order.number)
```

Comment:
It would have been better to add these new translation keys to `en.yml` within the same commit.

Reply:
Alright, I will rewrite the commit history.

Reply:
It's okay, not worth the hassle. I just mentioned it for next time. It's not a big deal.

Reply:
Oh, alright @mkllnk. Thanks for the tip!